src/sass/components/_header.scss

```diff
@@ -222,3 +225,32 @@ header {
     }
   }
 }
+// task 11 section--products
+.section--products {
+  .row {
+    justify-content: center;
+  }
+
+  .col-3 {
+    min-width: 280px;
+  }
+}
+
+.section--blog-posts {
+  .row {
+    justify-content: center;
+  }
+
+  .col-4 {
```

Comment:
nie zmieniamy szerokości kolumn grida - znajdź inne rozwiązanie na to co chcesz zrobić